www/js/views/login.js

```diff
@@ -65,11 +64,7 @@
                         FMS.isLoggedIn = 0;
                         that.model.set('password', '');
                         that.model.save();
-                        that.$('#form_email').val('');
-                        that.$('#form_password').val('');
-                        that.$('#success_row').hide();
-                        that.$('#signed_in_row').hide();
-                        that.$('#password_row').show();
+                        that.rerender();
```

Comment:
Similarly, the "Signed out!" message is never shown any more if I've read this right, and could be removed (it'd be nice to show a signed out message though of some sort on the sign-in screen in this case).